src/main/java/io/vinyldns/java/model/record/set/ListRecordSetChangeRequest.java

```diff
@@ -0,0 +1,74 @@
+package io.vinyldns.java.model.record.set;
```

Comment:
We don't have anything to automate headers yet, so would you be able to add the copyright text (eg. from the top of `ListBatchChangesRequest.java`) to the top of new files?